CTFd/admin/users.py

```diff
@@ -42,6 +42,20 @@ def users_listing():
                 .order_by(Users.id.asc())
                 .all()
             )
+        elif field == "ip":
+            # Find all the matching user IDs from the tracking table
+            matched_users = (
+                db.session.query(Tracking.user_id, Tracking.ip)
+                .filter(Tracking.ip.like("%{}%".format(q)))
+                .subquery()
```

Comment:
Would this be better as a JOIN?

Reply:
Not sure on how to go about this. Can you link a resource? :)